<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
 <layout class="QVBoxLayout" name="verticalLayout">
  <item>
   <layout class="QHBoxLayout" name="colorRowLayout">
     <item>
      <widget class="QLabel" name="backgroundColorLabel">
       <property name="text">
        <string>Background color:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hexEdit">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>90</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="maxLength">
        <number>7</number>
       </property>
       <property name="placeholderText">
        <string>#RRGGBB</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="chooseButton">
       <property name="text">
        <string>Chooseâ€¦</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="rowStretch">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
    </item>
   </layout>
  </item>
  <item>
   <widget class="QFrame" name="previewFrame">
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>64</width>
      <height>32</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
   </widget>
  </item>
  <item>
   <spacer name="bottomStretch">
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
    <property name="sizeHint" stdset="0">
     <size>
      <width>20</width>
      <height>40</height>
     </size>
    </property>
   </spacer>
  </item>
 </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
